release_notes/ocp-4-7-release-notes.adoc

```diff
@@ -40,6 +40,41 @@ This release adds improvements related to the following components and concepts.
 [id="ocp-4-7-rhcos"]
 === {op-system-first}
 
+[id="ocp-4-7-enhanced-disk-provisioning"]
+==== Enhanced disk provisioning for LUKS and RAID devices
```

Comment:
"LUKS and RAID devices and FBA DASD storage"?  "LUKS, RAID, and FBA DASD devices" doesn't sound right because the first two are virtual and the last is physical.  Or we could just simplify to "Enhanced disk provisioning".

Reply:
How about, "Enhanced disk provisioning for LUKS, RAID, and FBA DASD"?

Reply:
Sure.